doc/convertAll.md

```diff
@@ -0,0 +1,58 @@
+@function can-type/convertAll convertAll
+@parent can-type/methods 8
+@description Derive a new type that makes every property of Type convert, eliminating strict type checking.
+
+@signature `type.convertAll(Type)`
+
+  Given a `Type`, creates a [can-type.typeobject] where each property on the Type is converted, rather than being strictly checked.
```

Comment:
Is showing the `typeobject` as the type of the `@return` below enough? I think it makes it more confusing to mention it here also. I think saying something like the `type.all` description makes it more clear:

> Create a new type that inherits the same properties of Type, but where each property on the Type is converted, rather than being strictly checked.

Reply:
I did this.